Makefile

```diff
@@ -25,6 +25,12 @@ proto:
 	for dir in $$(git ls-files '*.proto' | xargs -n1 dirname | uniq); do \
 		protoc -I $$dir --go_out=plugins=grpc:$$dir $$dir/*.proto; \
 	done
+	cat grpclb/grpc_lb_v1/load_balancer.pb.go | \
```

Comment:
> Why would we do that, given that this work is already done, and is correct?

"Done and correct" is not the bar for submitting code, though.  This approach causes a couple maintenance problems for us:

1. `grpclb.proto` is now no longer a copy of the [canonical version](https://github.com/grpc/grpc/blob/master/src/proto/grpc/lb/v1/load_balancer.proto) of the proto.  This means updating it requires extra steps.
2. `load_balancer.pb.go` is modified by `sed`, which is not ideal for several reasons (e.g. it's not based on the AST like `gofmt` would be).

I can think of two low-effort options that would allow us to remove the `sed` commands until we can switch to using `go generate` in a later PR:

1. Leave the rest of the changes as-is, but skip the `grpclb.proto` file in `make proto` and delete `load_balancer.proto`.
2. Remove the service from `grpclb.proto` and re-add `grpclb/grpclb_server_generated.go`.

Either of those would be acceptable as a short/medium-term solution to our concerns.